test/RecipeRepository-test.js

```diff
@@ -1,8 +1,52 @@
 import { expect } from 'chai';
 import RecipeRepository from '../src/classes/RecipeRepository';
+import { recipeData } from '../src/data/recipes';
+import { ingredientsData } from '../src/data/ingredients'
 
 describe('Recipe', () => {
+  let repo;
+
+  beforeEach(() => {
+    repo = new RecipeRepository(recipeData);
+  });
+
   it('Should be a function', () => {
     expect(RecipeRepository).to.be.a('function');
   });
-})
\ No newline at end of file
+  
+  it('Should be an instance of Recipe', () => {
+    expect(repo).to.be.an.instanceOf(RecipeRepository);
+  });
+
+  it('Should have all recipes as properties', () => {
+    expect(repo[0].id).to.equal(595736);
+  });
+
+  it('Should filter recipes based on one tag', () => {
+    const tag = 'snack';
+    const expected = repo.findRecipesByTag(tag);
+    expect(expected.length).to.deep.equal(9);
```

Comment:
Would it make sense to add another expect statement that checks whether expected[0].name equals the name of recipe we expect?